app/models/peep.rb

```diff
@@ -0,0 +1,13 @@
+require 'data_mapper'
+
+class Peep
+
+  include DataMapper::Resource
+
+  property :id,        Serial
+  property :message,   Text
+  property :username,  String
+  property :name,      String
+  property :time_date, DateTime, :default => lambda { |p,s| DateTime.now.strftime("%a %d %B %Y - %H:%M") }
```

Comment:
Unused block argument - `p`. If it's necessary, use `_` or `_p` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Unused block argument - `s`. If it's necessary, use `_` or `_s` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Line is too long. [106/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.